### PR TITLE
Update robots.txt 

### PR DESCRIPTION
**Pull request by SandroLinux**
@EnderDev You forgot one of the three letter agencies

### CODE REVIEW

File: public/robots.txt [hunk 1]
```diff
@@ -5,4 +5,10 @@ User-agent: GCHQ
...
 
 User-agent: CIA
-Disallow: /
\ No newline at end of file
+Disallow: /
+
+User-agent: NSA
+Disallow: /
+
+User-agent: Rickrolling
```
Comment (EnderDev) [inline public/robots.txt:13]: Could we remove this?

**Review by EnderDev — CHANGES_REQUESTED** [review]

File: public/robots.txt (hunk 1 above)
Comment (SandroLinux) [reply public/robots.txt:13]: Sorry lol

Comment (SandroLinux) [reply public/robots.txt:13]: remove rickrolling?

**Review by SandroLinux — COMMENTED** [review]
I added FBI removed rickrolling

**Pull request merged**